openmetadata-docs/content/deployment/upgrade/kubernetes.md

```diff
@@ -27,18 +27,24 @@ Below document is valid for upgrading Helm Charts from **0.12.x to 0.13.0**.
 
 ### Back up metadata
 
-Before proceeding, pleae make sure you made a backup of your MySQL/Postgres DB behind OpenMetadata server. This step is extremely important for you to restore to your current state if any issues come up during the upgrade 
-
-<InlineCalloutContainer>
-  <InlineCallout
-    color="violet-70"
-    icon="luggage"
-    bold="Backup Metadata"
-    href="/deployment/backup-restore-metadata"
-  >
-    Learn how to back up MySQL data.
-  </InlineCallout>
-</InlineCalloutContainer>
+Before proceeding, pleae make sure you made a backup of your MySQL/Postgres DB behind OpenMetadata server. This step is extremely important for you to restore to your current state if any issues come up during the upgrade. It is recommended before upgrading you production instances.
+
+Make sure you have a connectivity between your database (mysql / postgresql) with host machine where you will be running the below commands. If you are using the default database available with OpenMetadata Dependencies, make sure to port-forward the mysql service using `kubectl port-forward service/mysql 3306:3306`.
+
+1. `python -m venv venv`
+2. `source venv/bin/activate`
+3. `PIP_USER=false pip install openmetadata-ingestion==0.13.0.3`
```

Comment:
```suggestion
3. `pip install openmetadata-ingestion~=0.13.1`
```

Backup fixes will be there for 0.13.1

Reply:
In that cases, should it be 0.13.1.0 as we are following the pattern X.X.X.X ?